core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/DefaultExchangesCreator.java

```diff
@@ -171,17 +196,25 @@ protected void doCreateExchange(FunctionalExchange functionalExchange_p, Compone
     }
 
     // Retrieve (or create) ports and set bounds of connections
-    ComponentPort outP = getRelatedProvidingPort(exchangeOutput_p, functionalExchange_p.getSource(), functionalExchange_p);
-    ComponentPort inP = getRelatedRequiringPort(exchangeInput_p, functionalExchange_p.getTarget(), functionalExchange_p);
+    ComponentPort outP = getRelatedProvidingPort(exchangeOutput_p, functionalExchange_p.getSource(),
+        functionalExchange_p);
+    ComponentPort inP = getRelatedRequiringPort(exchangeInput_p, functionalExchange_p.getTarget(),
+        functionalExchange_p);
     ce.setSource(outP);
     ce.setTarget(inP);
     ComponentExchangeExt.attachToDefaultContainer(ce);
 
     // Creates the exchange allocation
-    ComponentExchangeFunctionalExchangeAllocation cfea = FaFactory.eINSTANCE.createComponentExchangeFunctionalExchangeAllocation();
+    ComponentExchangeFunctionalExchangeAllocation cfea = FaFactory.eINSTANCE
+        .createComponentExchangeFunctionalExchangeAllocation();
     cfea.setSourceElement(ce);
     cfea.setTargetElement(functionalExchange_p);
     ce.getOwnedComponentExchangeFunctionalExchangeAllocations().add(cfea);
+
+    Status status = new Status(IStatus.INFO, Activator.PLUGIN_ID,
+        "The Component exchange " + ce.getName() + " has been succefully created between the exchange input component "
```

Comment:
Can you make information message navigable towards the elements 
(using EmbeddedMessage in the status like below: https://github.com/eclipse/capella/blob/5dfdd588f90ee4bac0cd803e47d3bf065be809e4/core/plugins/org.polarsys.capella.core.projection.interfaces/src/org/polarsys/capella/core/projection/interfaces/generateInterfaces/Rule_FunctionPort.java#L93